### PR TITLE
Update the deb packaging script to make unstable fetch versions from the CI

### PR DESCRIPTION
**Pull request by luxas**
In the future, we can consider a "nightly" channel as well.

We should just incorporate the rpm packaging process in this script or something later
cc @dgoodwin 

We should still bump the stable versions manually after we've validated them.

PTAL @mikedanese

### CODE REVIEW

**Comment by mikedanese** [comment]
This is awsome. Thanks!

File: debian/build.go [hunk 1]
```diff
@@ -175,6 +183,38 @@ func walkBuilds(builds []build, f func(pkg, distro, arch string, v version) erro
 	return nil
 }
 
+func fetchVersion(url, fallback string) string {
```
Comment (mikedanese) [inline debian/build.go]: fallback seems like a bad idea. If I set this up in CI how do I know whether by pushing the release I would be reverting to an old version? Just fail if we can't fetch a new version.

**Comment by idvoretskyi** [comment]
> In the future, we can consider a "nightly" channel as well.

@luxas as a product consumer, I'd like to have it. Thank you for the initiative.

**Comment by luxas** [comment]
@mikedanese @idvoretskyi Updated, now everything's fixed.
This is really cool, I hope @mikedanese will hook it up with his CI soon so I can start using it!

File: debian/build.go [hunk 2]
```diff
@@ -175,6 +198,56 @@ func walkBuilds(builds []build, f func(pkg, distro, arch string, v version) erro
...
+	res.Body.Close()
+	if err != nil {
+		return "", err
+	}
+	// Remove a newline and the v prefix from the string
+	return strings.Replace(strings.Replace(string(versionBytes), "v", "", 1), "\n", "", 1), nil
+}
+
+func getStableKubeVersion() (string, error) {
+	return fetchVersion("https://storage.googleapis.com/kubernetes-release/release/stable.txt")
```
Comment (ixdy) [inline debian/build.go]: https://dl.k8s.io/release/stable.txt is an alias for this

File: debian/build.go [hunk 3]
```diff
@@ -175,6 +198,56 @@ func walkBuilds(builds []build, f func(pkg, distro, arch string, v version) erro
...
+}
+
+func getLatestKubeVersion() (string, error) {
+	return fetchVersion("https://storage.googleapis.com/kubernetes-release/release/latest.txt")
```
Comment (ixdy) [inline debian/build.go]: likewise https://dl.k8s.io/release/latest.txt

**Comment by luxas** [comment]
@mikedanese Fixed @ixdy's nits now. PTAL

**Pull request merged**